presto-orc/src/main/java/com/facebook/presto/orc/OrcSelectiveRecordReader.java

```diff
@@ -188,7 +188,8 @@ public OrcSelectiveRecordReader(
             StripeMetadataSource stripeMetadataSource,
             boolean cacheable,
             RuntimeStats runtimeStats,
-            Optional<OrcFileIntrospector> fileIntrospector)
+            Optional<OrcFileIntrospector> fileIntrospector,
+            Map<String, Boolean> orcReaderUserOptions)
```

Comment:
Please no random maps with user options, everything related to options should go to OrcRecordReaderOptions

Reply:
Review comment fixed